update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -35,7 +35,7 @@ public static void setAdaptors(MySQLAdaptor adaptorTR, MySQLAdaptor adaptorGK) {
 	}
 
 	@SuppressWarnings("unchecked")
-	public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pathToReport, String releaseNumber, boolean testMode) throws IOException {
+	public static void findAndUpdateDOIs(long personId, String pathToReport, String releaseNumber, boolean testMode) throws IOException {
```

Comment:
Should `releaseNumber` be a typed as an `int`?

Reply:
If you follow how releaseNumber is used, its actually just to generate the 'doisToBeUpdated-vXX' file that is meant to be sent to curators automatically, so keeping it as a String makes sense. Maybe a better way of doing it would be to generate the path and then pass that as an argument?

Reply:
Okay, I see.  I'd still be inclined to make it an `int`.  Java will handle it properly when concatenated to a String and it makes it ensures a number is passed as the Reactome version.

Reply:
Or rename the variable to something that suggests more than just a number?